src/popup/router/components/Modals/SignModal.scss

```diff
@@ -6,6 +6,7 @@
   overflow-y: scroll;
```

Comment:
won't changing this to
```suggestion
  overflow-y: auto; // or even removing this
```
will work the same?

Reply:
Yes, you are right